src/Api/Endpoints/ProductEndpoints.cs

```diff
@@ -56,11 +56,12 @@ private async Task<IResult> AddProduct([FromBody] AddProductRequest request, IMe
         }
         catch (ValidationException validationException)
         {
+            logger.LogError("{ErrorMessage} {Errors}", validationException.Message, validationException.Errors);
```

Comment:
Perhaps it's not an error in our API if somebody sends invalid body in their request?
```suggestion
            logger.LogInformation("{ErrorMessage} {Errors}", validationException.Message, validationException.Errors);
```

Reply:
Nice question, it depends on how do we want to log it.

Reply:
Finally, I decided to switch to the Information way.